mailbox/store/src/main/java/org/apache/james/mailbox/store/ResultUtils.java

```diff
@@ -100,17 +100,17 @@ public static MessageResult loadMessageResult(MailboxMessage message, FetchGroup
             if (fetchGroup != null) {
                 int content = fetchGroup.content();
 
-                if ((content & FetchGroup.HEADERS) > 0) {
-                    content -= FetchGroup.HEADERS;
+                if ((content & FetchGroup.HEADERS_MASK) > 0) {
```

Comment:
all those byte operation `&` and `-` should be domain method in the `FetchGroup` class

Reply:
Let's take this refactoring another step further then :-)